test/end-to-end/cases/focus-vs-pending-attribute.xspec

```diff
@@ -0,0 +1,92 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+	focus-vs-pending-element.xspec copies this file.
+	When modifying this file, check whether focus-vs-pending-element.xspec needs any additions or
+	updates.
+-->
+<x:description query="x-urn:test:do-nothing" query-at="do-nothing.xqm" schematron="do-nothing.sch"
+	stylesheet="do-nothing.xsl" xml:base="../../" xmlns:x="http://www.jenitennison.com/xslt/xspec">
+
+	<x:scenario label="Expect" shared="yes">
+		<x:expect label="Always Success" test="true()" />
+	</x:scenario>
+
+	<x:scenario label="/Neutral">
+		<x:call function="true" />
+		<x:like label="Expect" />
+	</x:scenario>
+
+	<x:scenario label="/Neutral">
+		<x:call function="true" />
+		<x:like label="Expect" />
+
+		<x:scenario label="/Neutral">
+			<x:like label="Expect" />
+		</x:scenario>
+
+		<x:scenario focus="Focus #1" label="/Focued #1">
```

Comment:
Typo: Focued (here and in `focus-vs-pending-element.xspec`)

Reply:
Fixed in 7314baa5893a266504e27ec4653e6f36b4f5e626